### PR TITLE
Add April planning session notes

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: governance/coordination-structure.md [hunk 1]
```diff
@@ -22,6 +22,22 @@ Toronto Mesh uses voting to determine whether we have reached consensus:
...
+
+| Working Group | Lead |
+| --- | --- |
+| **Central Org** (_ongoing_) | dasanchez |
+| (Welcome/Intros) **Onboarding** (_ongoing_) | Pedro.S |
+| **Tools** (_ongoing_) | co-lead benhylau, dasanchez, Pedro.S, udit |
+| **Website** (_ongoing_) | dcwalk |
+| **Virtual Mesh** (_ongoing_) |  |
+| **Deployment** (_ongoing_) | Pedro.S |
+| **node.tech** (_ongoing_) | benhylau |
```
Comment (benhylau) [inline governance/coordination-structure.md]: udit

File: governance/coordination-structure.md [hunk 2]
```diff
@@ -22,6 +22,22 @@ Toronto Mesh uses voting to determine whether we have reached consensus:
 
 ## Working Groups
 
+### Cycle IV (2017 April)
+
+Third working groups with leads established at November 12 Planning Meeting.
+Excerpted from [Planning Meeting Notes - November 12, 2016](https://github.com/tomeshnet/documents/blob/master/meeting_notes/20161112_planning-meeting-notes.md)
```
Comment (dasanchez) [inline governance/coordination-structure.md]: Should L27 and L28 list the April date & meeting notes?

**Review by dasanchez — CHANGES_REQUESTED** [review]
Cycle IV working groups were established in April

File: meeting_notes/20170424_planning-meeting-notes.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+1. apply for makerfest
+1. restart conversations with TPL
+1. bi-weekly node.tech meetups over the summer
+1. public hardware workshops
+1. map up on our website
+1. test network (2 nodes with physical link >100m)
+
+_Who needs to be added to what services?_ (Tools working group)
+	- hello@tomesh.net
```
Comment (benhylau) [inline meeting_notes/20170424_planning-meeting-notes.md]: Indent

**Comment by dcwalk** [comment]
Fixed, thanks for the catch!

**Pull request merged**